app/src/main/java/org/mozilla/fenix/library/history/HistoryFragmentStore.kt

```diff
@@ -131,7 +136,8 @@ sealed class HistoryFragmentAction : Action {
 data class HistoryFragmentState(
     val items: List<History>,
     val mode: Mode,
-    val pendingDeletionIds: Set<Long>,
+    val pendingDeletionItems: Set<PendingDeletionHistory>,
+    val isEmpty: Boolean,
```

Comment:
Could we determine the if is empty from the `items`?

Reply:
Sadly, nope. Items inside the `HistoryFragmentState` are empty. Adapter is being fed data directly, avoiding the state:
```
lifecycleScope.launch {
            history.collect {
                historyView.historyAdapter.submitData(it)
            }
        }
```